e2e/test/buttons-tests/buttons-test.js

```diff
@@ -30,6 +31,16 @@ describe('FEF buttons tests', () => {
        '"Yeah dropdown" is not similar to the reference');
   });
 
+  it('should check the button with mouseover: ' + YEAH_DROPDOWN_NAME, () => {
+    browser.url('/buttons');
+    $(SAVE_BUTTON_SELECTOR).scroll();
+    browser.moveToElement('.dropdown');
```

Comment:
it's hard to follow this test it uses `YEAH_DROPDOWN_NAME`, `SAVE_BUTTON_SELECTOR` and even the selector of `.dropdown`